### PR TITLE
fix: normal-severity action not registered

### PR DESCRIPTION
**Pull request by RomainMuller**
When submitted as ARNs instead of IAlarmAction, normal
severity alarm actions were not bound to the alarms as
they should have.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: src/monitoring/index.ts [hunk 1]
```diff
@@ -75,7 +75,10 @@ export class Monitoring extends Construct implements IMonitoring {
 
   public addLowSeverityAlarm(_title: string, alarm: cw.Alarm) {
     if (this.alarmActions?.normalSeverityAction) {
-      alarm.addAlarmAction(this.alarmActions!.normalSeverityAction);
+      alarm.addAlarmAction(this.alarmActions.normalSeverityAction);
+    }
+    if (this.alarmActions?.normalSeverityAction) {
```
Comment (eladb) [inline src/monitoring/index.ts:80]: Now it seems like this is adding the normal severity action twice.

Also - missing test?

**Review by eladb — CHANGES_REQUESTED** [review]

File: src/monitoring/index.ts [hunk 2]
```diff
@@ -74,8 +74,13 @@ export class Monitoring extends Construct implements IMonitoring {
   }
 
   public addLowSeverityAlarm(_title: string, alarm: cw.Alarm) {
-    if (this.alarmActions?.normalSeverityAction) {
-      alarm.addAlarmAction(this.alarmActions!.normalSeverityAction);
+    const normalSeverityActionArn = this.alarmActions?.normalSeverity;
+    if (normalSeverityActionArn) {
+      alarm.addAlarmAction({ bind: () => ({ alarmActionArn: normalSeverityActionArn }) });
+    }
+    const highSeverityAction = this.alarmActions?.normalSeverityAction;
```
Comment (eladb) [inline src/monitoring/index.ts:81]:
```suggestion
    const highSeverityAction = this.alarmActions?.highSeverityAction;
```

File: src/__tests__/monitoring.test.ts [hunk 3]
```diff
@@ -147,4 +148,28 @@ describe('web canary handler', () => {
     process.env.URL = 'https://amazon.com/not-found-please12345';
     await expect(webCanaryHandler({})).rejects.toThrow(/Response code 404 \(Not Found\)/);
   });
-});
\ No newline at end of file
+});
+
+test('low-severity alarm actions are registered', () => {
```
Comment (eladb) [inline src/__tests__/monitoring.test.ts:153]: What about high severity?

Isn't this called "normal severity"?

**Review by eladb — CHANGES_REQUESTED** [review]

File: src/__tests__/monitoring.test.ts (hunk 3 above)
Comment (RomainMuller) [reply src/__tests__/monitoring.test.ts]: > What about high severity?

I don't normally get in the business of adding missing test coverage on code paths unrelated to my changes... but I've added that now.

> Isn't this called "normal severity"?

Ah yes! Fixed.

Comment (eladb) [reply src/__tests__/monitoring.test.ts]: > I don't normally get in the business of adding missing test coverage on code paths unrelated to my changes... but I've added that now.

If possible, why not?

**Review by eladb — APPROVED** [review]

File: src/__tests__/monitoring.test.ts (hunk 3 above)
Comment (RomainMuller) [reply src/__tests__/monitoring.test.ts]: To keep the diff small & focused.

**Pull request merged**